### ISSUE TITLE
Upgrade msal-angular to use msal-core 1.x

### ISSUE THREAD

**Issue by jasonnutter** [AzureAD/microsoft-authentication-library-for-js#1033]
Currently, `msal-angular` uses `msal` version `0.2.2`, which is a major version behind and is not currently receiving any bug fixes or new features. 

AC:
1. `msal-angular` uses the current major version of `msal`.

**Comment by jasonshsoftware** [comment]
Is there any update on this? I am trying to determine if I should delay my deployment until this is completed. We need this update because MSAL 0.2.2 has a hardcoded prompt=select_account in the url.

**Comment by Ivan-L** [comment]
Hi @jasonnutter, I'm just adding our +10 to this.

I've literally been frequenting this repository at least twice a week for the last few months in anticipation of MSAL Angular 1.0. We've got a project in production running on msal-angular 0.1.4 so the sooner we get to 1.0 the better.

Thanks!

**Comment by jasonnutter** [comment]
Update: this has been pushed back due to other priority work, but we're still committed to have this done this month.

**Comment by jasonnutter** [comment]
@Ivan-L Thanks for the feedback! I'll update this issue when there is a version available you can try. So you can subscribe to the issue to get automatically notified, so that you don't have to check manually :)

**Comment by JakobGSvendsen** [comment]
We are in the same boat as Ivan-L. We are really looking forward to this, and hope this will not be postponed much more. There was critical stuff like browser support (edge) that isnt in the old version and state is not supported which make reply URL management much worse.. So this is really needed! Looking forward to anything we can try ;-)

**Comment by jasonnutter** [comment]
The first version of MSAl Angular that uses the latest MSAL is available: `@azure/msal-angular@1.0.0-alpha.0`

Please try this version and let us know of any feedback! Note, this is pre-release version intended to gather feedback, not for production use. Thanks!

**Comment by raboud** [comment]
@jasonnutter is the source for that in the dev-angular-1.0-msal-1 branch. I noticed a bunch of documentation errors and errors in the samples all under the msal-angular stuff in that branch. I am more than willing to update them all but and not sure how to put in a pull request against a branch. Is that even the right way to attack the problem. It seems with the new way to configure the service the documentation and samples did not get updated.

**Comment by jasonnutter** [comment]
@raboud Correct, and it can be tracked in #1160. I did update the main MSAL Angular sample and some of the docs in the PR, so can you please clarify exactly what isn't working? Thanks!

**Comment by raboud** [comment]
https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev-angular-1.0-msal-1/lib/msal-angular/tests/sample/src/app/app.module.ts

`   MsalModule.forRoot({
   	    clientID: '79d1dd3f-4de3-4b69-ac25-f2fc5eefe773',
        authority: "https://login.microsoftonline.com/msidlab4.onmicrosoft.com/",
        validateAuthority: true,
        redirectUri: "https://msalangularsample.azurewebsites.net/",
        cacheLocation : "localStorage",
        postLogoutRedirectUri: "https://msalangularsample.azurewebsites.net",
        navigateToLoginRequestUrl: true,
        popUp: false,
        unprotectedResources: ["https://www.microsoft.com/en-us/"],
        protectedResourceMap: protectedResourceMap,
        logger: loggerCallback,
        correlationId: '1234',
        level: LogLevel.Info,
        piiLoggingEnabled: true
      }`

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev-angular-1.0-msal-1/lib/msal-angular/README.md

 MsalModule.forRoot

**Comment by jasonnutter** [comment]
@raboud Ah, gotya, thanks! For the sample app you linked, there's a good chance it will get removed and replaced, which is why I didn't update it, so I'll address that this week. The main MSAL Angular sample has been updated to show the new usage, which you can see here: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev-angular-1.0-msal-1/samples/MSALAngularDemoApp/src/app/app.module.ts#L41-L66

And I'll make sure the README gets updated, I missed that. Thanks!

**Comment by raboud** [comment]
Do you need any help updated to a later cli or update the rxjs we dont need the rxjs-compa module?

**Comment by jasonnutter** [comment]
@raboud Not at this time. I will need help testing once it is ready, though!
